assets/src/routes/routes.js

```diff
@@ -1,41 +1,45 @@
 import { Grade } from '@material-ui/icons'
 
-const routes = (courseId, activeViews) => {
+const routes = (courseId, activeViews, activeOnly=true) => {
   const allViews = [
     {
       path: `/courses/${courseId}/resources`,
       title: 'Resources Accessed',
       icon: Grade,
       description: 'See what resources you and your peers are viewing.',
       image: '/static/images/file_access_trends_icon.png',
```

Comment:
Not part of this PR, but I just wanted to say it'll be nice to reorganize the backend to include models for the various visualizations that includes things like title, description, icon, image, etc.  Visualizations could be treated as plug-in modules.

Reply:
@lsloan, if this is something you want to happen, I'd suggest creating an issue; no one will remember the suggestion if it only lives here. I'm resolving this for now.